File: 2.cpp
//link: http://codeforces.com/problemset/problem/4/A


#include <iostream>
using namespace std;

int main(void)
{
  int w;
  bool flag = true;

  cin>>w;

  if (w <= 2 || w%2!=0)
  {
    flag = false;
    cout<<"NO";
  }
  else
  {
    for (int i=2; i<=w/2; i+=2)
    {
      if ((w-i)%2 == 0)
      {
        cout<<"YES";
        flag = false;
        break;
      }
    }
  }

  if (flag)
    cout<<"NO";
}

File: 2.py
# http://codeforces.com/problemset/problem/4/A

w = int(input())

if w<=2 or w%2!=0:
  print("NO")
else:
  print("YES")
  # for i in range(2, int(w/2), 2):
  #   if w-i % 2 == 0:
  #     print("YES")
  #     break